rs/recovery/src/lib.rs

```diff
@@ -913,10 +913,14 @@ pub async fn get_node_metrics(logger: &Logger, ip: &IpAddr) -> Option<NodeMetric
         let mut parts = line.split(' ');
         if let (Some(prefix), Some(height)) = (parts.next(), parts.next()) {
             match prefix {
-                "certification_last_certified_height" => match height.trim().parse::<u64>() {
-                    Ok(val) => node_heights.certification_height = Height::from(val),
-                    error => warn!(logger, "Couldn't parse height {}: {:?}", height, error),
-                },
+                r#"artifact_pool_certification_height_stat{pool_type="validated",stat="max",type="certification"}"# => {
```

Comment:
Nice, thanks! Would it be possible to display the height of the latest certification share, as well?